Reading List/Reading List/ReadingListTableViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ReadingListTableTableViewController.swift
+//  Reading List
+//
+//  Created by James Sedlacek on 6/5/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class ReadingListTableViewController: UITableViewController {
+    
+    let bookController: BookController = BookController()
+
+    override func viewDidLoad() {
+        bookController.loadFromPersistentStore()
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        bookController.loadFromPersistentStore()
+        tableView.reloadData()
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 2 //read and unread books
+    }
+
+    override func tableView(_ tableView: UITableView,
+    numberOfRowsInSection section: Int) -> Int {
+        if section == 0 {
+            return bookController.readBooks.count
+        } else {
+            return bookController.unreadBooks.count
+        }
+    }
+    
+    private func bookFor(indexPath: IndexPath) -> Book {
+        if indexPath.section == 0 {
+            return bookController.readBooks[indexPath.row]
+        } else {
+            return bookController.unreadBooks[indexPath.row]
+        }
+    }
+    
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "bookCell", for: indexPath) as? BookTableViewCell else { return UITableViewCell()}
+
+        cell.delegate = self
+        if indexPath.section == 0 {
+            if bookFor(indexPath: indexPath).hasBeenRead {
+                cell.bookTitleLabel.text = bookFor(indexPath: indexPath).title
+                cell.checkbox.setImage(UIImage(named: "checked"), for: .normal)
+            }
+        } else {
+            if !bookFor(indexPath: indexPath).hasBeenRead {
+                cell.bookTitleLabel.text = bookFor(indexPath: indexPath).title
+                cell.checkbox.setImage(UIImage(named: "unchecked"), for: .normal)
+            }
+        }
```

Comment:
This would probably be better handled in the cell itself by using a didSet on the book to update the views, and just setting the book property here in the TVC